Tests/TuistGeneratorTests/GraphMappers/AutogeneratedProjectSchemeWorkspaceMapperTests.swift

```diff
@@ -110,10 +100,51 @@ final class AutogeneratedProjectSchemeWorkspaceMapperTests: TuistUnitTestCase {
                 ),
             ])
         )
+        XCTAssertFalse(try XCTUnwrap(scheme.testAction?.coverage))
     }
+    
+    func test_code_coverage_config() throws {
+        // Given
+        let subject = AutogeneratedProjectSchemeWorkspaceMapper(enableCodeCoverage: true)
+        let targetA = Target.test(
+            name: "A"
+        )
+        let targetATests = Target.test(
+            name: "ATests",
+            product: .unitTests,
+            dependencies: [.target(name: "A")]
+        )
+
+        let projectPath = try temporaryPath()
+        let project = Project.test(
+            path: projectPath,
+            targets: [
+                targetA,
+                targetATests,
+            ]
+        )
+
+        let workspace = Workspace.test(
+            name: "A",
+            projects: [
+                project.path,
+            ]
+        )
+
+        // When
+        let (got, sideEffects) = try subject.map(
+            workspace: WorkspaceWithProjects(workspace: workspace, projects: [project])
+        )
+
+        // Then
+        let scheme = try XCTUnwrap(got.workspace.schemes.first)
+        XCTAssertTrue(try XCTUnwrap(scheme.testAction?.coverage))
```

Comment:
```suggestion
        XCTAssertTrue(scheme.testAction?.coverage)
```

Reply:
Facepalm :D thanks for the suggestion here!

Reply:
Rebased, fixed the style issue and applied the suggestion